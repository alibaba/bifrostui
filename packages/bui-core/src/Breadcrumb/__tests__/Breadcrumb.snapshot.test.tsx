import { snapshotTest } from 'testing';

describe('Breadcrumb snapshot', () => {
  snapshotTest('Breadcrumb');
});
