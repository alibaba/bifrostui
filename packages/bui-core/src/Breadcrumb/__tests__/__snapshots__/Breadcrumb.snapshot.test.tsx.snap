// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Breadcrumb snapshot > Breadcrumb demo snapshot 0 1`] = `
<nav
  className="bui-breadcrumb"
>
  <ol
    className="bui-breadcrumb-list"
  >
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      首页
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      组件
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
    >
      面包屑
    </li>
  </ol>
</nav>
`;

exports[`Breadcrumb snapshot > Breadcrumb demo snapshot 0 2`] = `
<nav
  className="bui-breadcrumb"
>
  <ol
    className="bui-breadcrumb-list"
  >
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      首页
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      组件
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
    >
      面包屑
    </li>
  </ol>
</nav>
`;

exports[`Breadcrumb snapshot > Breadcrumb demo snapshot 0 3`] = `
<nav
  className="bui-breadcrumb"
>
  <ol
    className="bui-breadcrumb-list"
  >
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
      style={
        {
          "alignItems": "center",
          "display": "flex",
        }
      }
    >
      <span
        style={
          {
            "alignItems": "center",
            "display": "flex",
          }
        }
      >
        <svg
          className="bui-svg-icon"
          dangerouslySetInnerHTML={
            {
              "__html": "<path d="m56.7 10.4 28.7 24.2.5.4c3.5 3.3 3.9 8.7 1 12.4-.3.4-.7.8-1.2 1.1l-1 .6c-2.1 1.5-3.4 3.9-3.4 6.6v14.8c0 7.8-5.3 13.8-12.7 13.8H29.5c-7.6 0-14.5-6.2-14.5-13.8V54c0-2-1.2-3.8-3-4.6-1.1-.5-1.9-1-2.6-1.9-3-3.8-2.5-9.2.9-12.4l29-24.7c5-4 12.4-4 17.4 0zm-12.6 5.9L15.6 40.7c-.5.4-.7 1.2-.5 1.8 4.5 2 7.5 6.6 7.5 11.6v16.6c0 3.1 3.3 6.1 6.9 6.1h38.9c3.1 0 5.1-2.3 5.1-6.1V55.9c0-5.2 2.5-10 6.7-12.9l.7-.5.1-.3c.1-.6-.1-1.2-.5-1.6L51.9 16.4c-2.2-1.8-5.7-1.8-7.8-.1zm4 28c3 0 6.7 2.2 11.9 6.6 3 2.5 4.7 6.2 4.7 10.1v3.7c0 1.4-.7 2.6-1.9 3.3-1.2.7-2.6.7-3.8 0-1.2-.7-1.9-1.9-1.9-3.3V61c0-1.6-.7-3.2-2-4.2-2.1-1.8-3.9-3.1-5.3-3.9-.4-.2-.8-.5-1.3-.7l-.2-.2h-.2l-.3.1-.7.4-.7.4c-1.4.8-3.2 2.2-5.4 4-1.3 1-2 2.6-2 4.2v3.7c0 2.1-1.7 3.8-3.8 3.8s-3.8-1.7-3.8-3.8V61c0-3.9 1.7-7.6 4.8-10.1 5.4-4.4 9-6.6 11.9-6.6z" style="fill:#3c3f44"/>",
            }
          }
          focusable="false"
          viewBox="0 0 96 96"
        />
      </span>
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      <span
        style={
          {
            "alignItems": "center",
            "display": "flex",
          }
        }
      >
        <svg
          className="bui-svg-icon"
          dangerouslySetInnerHTML={
            {
              "__html": "<path d="M20 66c3.3 0 6 2.7 6 6v4c0 3.3-2.7 6-6 6h-4c-3.3 0-6-2.7-6-6v-4c0-3.3 2.7-6 6-6h4zm63 5c1.7 0 3 1.3 3 3s-1.3 3-3 3H37c-1.7 0-3-1.3-3-3s1.3-3 3-3h46zm-63 1h-4v4h4v-4zm0-32c3.3 0 6 2.7 6 6v4c0 3.3-2.7 6-6 6h-4c-3.3 0-6-2.7-6-6v-4c0-3.3 2.7-6 6-6h4zm63 5c1.7 0 3 1.3 3 3s-1.3 3-3 3H37c-1.7 0-3-1.3-3-3s1.3-3 3-3h46zm-63 1h-4v4h4v-4zm0-32c3.3 0 6 2.7 6 6v4c0 3.3-2.7 6-6 6h-4c-3.3 0-6-2.7-6-6v-4c0-3.3 2.7-6 6-6h4zm63 5c1.7 0 3 1.3 3 3s-1.3 3-3 3H37c-1.7 0-3-1.3-3-3s1.3-3 3-3h46zm-63 1h-4v4h4v-4z"/>",
            }
          }
          focusable="false"
          viewBox="0 0 96 96"
        />
        <span
          style={
            {
              "marginLeft": "4px",
            }
          }
        >
          组件
        </span>
      </span>
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
    >
      面包屑
    </li>
  </ol>
</nav>
`;

exports[`Breadcrumb snapshot > Breadcrumb demo snapshot 0 4`] = `
<nav
  className="bui-breadcrumb"
>
  <ol
    className="bui-breadcrumb-list"
  >
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      首页
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      &gt;
    </li>
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      组件
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      &gt;
    </li>
    <li
      className="bui-breadcrumb-item"
    >
      面包屑
    </li>
  </ol>
</nav>
`;

exports[`Breadcrumb snapshot > Breadcrumb demo snapshot 0 5`] = `
<nav
  className="bui-breadcrumb"
>
  <ol
    className="bui-breadcrumb-list"
  >
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      首页
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      :
    </li>
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      组件
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
    >
      面包屑
    </li>
  </ol>
</nav>
`;

exports[`Breadcrumb snapshot Breadcrumb demo snapshot 0 1`] = `
<nav
  className="bui-breadcrumb"
>
  <ol
    className="bui-breadcrumb-list"
  >
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      首页
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      组件
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
    >
      面包屑
    </li>
  </ol>
</nav>
`;

exports[`Breadcrumb snapshot Breadcrumb demo snapshot 0 2`] = `
<nav
  className="bui-breadcrumb"
>
  <ol
    className="bui-breadcrumb-list"
  >
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      首页
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      组件
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
    >
      面包屑
    </li>
  </ol>
</nav>
`;

exports[`Breadcrumb snapshot Breadcrumb demo snapshot 0 3`] = `
<nav
  className="bui-breadcrumb"
>
  <ol
    className="bui-breadcrumb-list"
  >
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
        }
      }
    >
      <span
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
          }
        }
      >
        <svg
          className="bui-svg-icon"
          dangerouslySetInnerHTML={
            Object {
              "__html": "<path d=\\"m56.7 10.4 28.7 24.2.5.4c3.5 3.3 3.9 8.7 1 12.4-.3.4-.7.8-1.2 1.1l-1 .6c-2.1 1.5-3.4 3.9-3.4 6.6v14.8c0 7.8-5.3 13.8-12.7 13.8H29.5c-7.6 0-14.5-6.2-14.5-13.8V54c0-2-1.2-3.8-3-4.6-1.1-.5-1.9-1-2.6-1.9-3-3.8-2.5-9.2.9-12.4l29-24.7c5-4 12.4-4 17.4 0zm-12.6 5.9L15.6 40.7c-.5.4-.7 1.2-.5 1.8 4.5 2 7.5 6.6 7.5 11.6v16.6c0 3.1 3.3 6.1 6.9 6.1h38.9c3.1 0 5.1-2.3 5.1-6.1V55.9c0-5.2 2.5-10 6.7-12.9l.7-.5.1-.3c.1-.6-.1-1.2-.5-1.6L51.9 16.4c-2.2-1.8-5.7-1.8-7.8-.1zm4 28c3 0 6.7 2.2 11.9 6.6 3 2.5 4.7 6.2 4.7 10.1v3.7c0 1.4-.7 2.6-1.9 3.3-1.2.7-2.6.7-3.8 0-1.2-.7-1.9-1.9-1.9-3.3V61c0-1.6-.7-3.2-2-4.2-2.1-1.8-3.9-3.1-5.3-3.9-.4-.2-.8-.5-1.3-.7l-.2-.2h-.2l-.3.1-.7.4-.7.4c-1.4.8-3.2 2.2-5.4 4-1.3 1-2 2.6-2 4.2v3.7c0 2.1-1.7 3.8-3.8 3.8s-3.8-1.7-3.8-3.8V61c0-3.9 1.7-7.6 4.8-10.1 5.4-4.4 9-6.6 11.9-6.6z\\" style=\\"fill:#3c3f44\\"/>",
            }
          }
          focusable="false"
          viewBox="0 0 96 96"
        />
      </span>
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      <span
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
          }
        }
      >
        <svg
          className="bui-svg-icon"
          dangerouslySetInnerHTML={
            Object {
              "__html": "<path d=\\"M20 66c3.3 0 6 2.7 6 6v4c0 3.3-2.7 6-6 6h-4c-3.3 0-6-2.7-6-6v-4c0-3.3 2.7-6 6-6h4zm63 5c1.7 0 3 1.3 3 3s-1.3 3-3 3H37c-1.7 0-3-1.3-3-3s1.3-3 3-3h46zm-63 1h-4v4h4v-4zm0-32c3.3 0 6 2.7 6 6v4c0 3.3-2.7 6-6 6h-4c-3.3 0-6-2.7-6-6v-4c0-3.3 2.7-6 6-6h4zm63 5c1.7 0 3 1.3 3 3s-1.3 3-3 3H37c-1.7 0-3-1.3-3-3s1.3-3 3-3h46zm-63 1h-4v4h4v-4zm0-32c3.3 0 6 2.7 6 6v4c0 3.3-2.7 6-6 6h-4c-3.3 0-6-2.7-6-6v-4c0-3.3 2.7-6 6-6h4zm63 5c1.7 0 3 1.3 3 3s-1.3 3-3 3H37c-1.7 0-3-1.3-3-3s1.3-3 3-3h46zm-63 1h-4v4h4v-4z\\"/>",
            }
          }
          focusable="false"
          viewBox="0 0 96 96"
        />
        <span
          style={
            Object {
              "marginLeft": "4px",
            }
          }
        >
          组件
        </span>
      </span>
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
    >
      面包屑
    </li>
  </ol>
</nav>
`;

exports[`Breadcrumb snapshot Breadcrumb demo snapshot 0 4`] = `
<nav
  className="bui-breadcrumb"
>
  <ol
    className="bui-breadcrumb-list"
  >
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      首页
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      &gt;
    </li>
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      组件
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      &gt;
    </li>
    <li
      className="bui-breadcrumb-item"
    >
      面包屑
    </li>
  </ol>
</nav>
`;

exports[`Breadcrumb snapshot Breadcrumb demo snapshot 0 5`] = `
<nav
  className="bui-breadcrumb"
>
  <ol
    className="bui-breadcrumb-list"
  >
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      首页
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      :
    </li>
    <li
      className="bui-breadcrumb-item"
      onClick={[Function]}
    >
      组件
    </li>
    <li
      className="bui-breadcrumb-item-separator"
    >
      /
    </li>
    <li
      className="bui-breadcrumb-item"
    >
      面包屑
    </li>
  </ol>
</nav>
`;
